scripts/release/constants.ts

```diff
@@ -30,3 +30,5 @@ export const GITHUB_REPOSITORY = process.env.GITHUB_REPOSITORY;
 export const DEFAULT_BRANCH = process.env.DEFAULT_BRANCH;
 export const NIGHTLY_BRANCH = process.env.NIGHTLY_BRANCH;
 export const PR_FILES_STARTS_WITH = ["packages/"];
+export const isNightlyRelease = isNightly(VERSION);
+export const AWAITING_RELEASE_LABEL = "awaiting release";
```

Comment:
Should we add this label to each PR that's merged to `dev`?

Reply:
just thought about the flow some more and i think this is what i'm gonna go for. this way if a PR is reverted we don't prematurely close anything (unless of course a label is incorrectly removed or added)

1. PR is merged
  - add "awaiting release" label to PR and linked issues
2. nightly release comments on PR and linked issues
3. stable comes around
  a. find merged PRs
  b. comment on PRs that have "awaiting release" label
    - remove "awaiting release" label
  c. comment on linked issues
    - remove "awaiting release" label

Reply:
Sounds good to me 👍

Reply:
actually, we don't need the label on issues, the PR label should be enough